ci-operator/step-registry/ipi/conf/azure/sharednetwork/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+- abhinavdahiya
```

Comment:
Can we extend this to [include][1] @fabianofranz and @jhixson74 ?  Having a single approver seems like a risk for over-silo'ed understanding (although root-ward approvers are also theoretically on the hook to provide maintenance in this subdir as well).

[1]: https://github.com/openshift/installer/blob/145e5ccdf32513ee96c777853e1a7a7620781960/OWNERS_ALIASES#L63-L65

Reply:
Out of band, @abhinavdahiya wants to punt on this too, like [here][1].

[1]: https://github.com/openshift/release/pull/9517#discussion_r438269592